src/node_file.cc

```diff
@@ -863,6 +868,26 @@ static void MKDir(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static void RealPath(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (args.Length() < 1)
+    return TYPE_ERROR("path is required");
+  if (!args[0]->IsString())
+    return TYPE_ERROR("path must be a string");
+
+  node::Utf8Value path(env->isolate(), args[0]);
```

Comment:
Indeed it should. Good catch.
On Apr 8, 2016 12:38 PM, "Trevor Norris" <notifications@github.com> wrote:

> In src/node_file.cc
> <https://github.com/nodejs/node/pull/3594#discussion_r59078332>:
>
> > @@ -863,6 +868,26 @@ static void MKDir(const FunctionCallbackInfo<Value>& args) {
> >    }
> >  }
> >
> > +static void RealPath(const FunctionCallbackInfo<Value>& args) {
> > +  Environment* env = Environment::GetCurrent(args);
> > +
> > +  if (args.Length() < 1)
> > +    return TYPE_ERROR("path is required");
> > +  if (!args[0]->IsString())
> > +    return TYPE_ERROR("path must be a string");
> > +
> > +  node::Utf8Value path(env->isolate(), args[0]);
>
> @jasnell <https://github.com/jasnell> This should probably accept the
> same set of objects as the other functions you updated, right?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/3594/files/4e1399d2524dfd361779dbd692d9bc4819f3b38d#r59078332>
>